port/PyAssimp/README.md

```
@@ -90,3 +90,23 @@ TEST
+BUILD AND DEPLOY
+----------------
+
+PyAssimp is a pure-Python package. Wheels do not bundle `libassimp`; users still
+need a matching Assimp shared library at runtime.
+
+Bump `version` in `pyproject.toml` before a release. PyPI does not allow
+replacing an existing version; yank and ship a new version if a bad release
+goes out.
+
+```console
+$ pip install -e ".[release]"
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)